### PR TITLE
add perf test case for state get http to compare http and grpc perfor…

### PR DESCRIPTION
**Pull request by skyao**
# Description

add perf test case for state get http to compare http and grpc performance.

## Issue reference

https://github.com/dapr/dapr/issues/4743

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by hueifeng** [comment]
Please fix DCO.

File: tests/dapr_tests.mk [hunk 1]
```diff
@@ -56,7 +56,7 @@ E2E_TESTAPP_DIR=./tests/apps
 PERF_TESTAPP_DIR=./tests/apps/perf
 
 # PERFORMANCE tests
-PERF_TESTS=actor_timer actor_reminder actor_activation service_invocation_http service_invocation_grpc state_get pubsub_publish
+PERF_TESTS=actor_timer actor_reminder actor_activation service_invocation_http service_invocation_grpc state_get state_get_http pubsub_publish
```
Comment (daixiang0) [inline tests/dapr_tests.mk:59]: Recommend to rename as `state_get_grpc` and `state_get_http`.

**Review by daixiang0 — DISMISSED** [review]
Codes look good overall, just small nits.

File: tests/dapr_tests.mk (hunk 1 above)
Comment (skyao) [reply tests/dapr_tests.mk:59]: OK

**Review by daixiang0 — APPROVED** [review]

**Pull request merged**